packages/@aws-cdk/cx-api/lib/artifacts/cloudformation-artifact.ts

```diff
@@ -5,6 +5,32 @@ import { CloudArtifact } from '../cloud-artifact';
 import { CloudAssembly } from '../cloud-assembly';
 import { Environment, EnvironmentUtils } from '../environment';
 
+/**
+ * Information needed to access an IAM role created
+ * as part of the bootstrap process
+ */
+export interface BootstrapRole {
```

Comment:
Actually I wanted this whole structure in the CloudAssembly itself. This module only needs to read it.